templates/home/base.html

```diff
@@ -16,69 +17,88 @@
-            {% else %}
-            <li class="nav-item mr-2 col-md-3 col-sm-4" data-toggle="collapse"
-                data-target=".navbar-collapse.show">
-                <a href="{% url 'landing-home' %}" class="nav-link nav-text-color book-button hvr-box-shadow-inset">
-                    <i class="fas fa-play  fa-lg"></i> Sign Up</a>
-            </li><li class="nav-item mr-2 col-md-3 col-sm-4" data-toggle="collapse"
-            data-target=".navbar-collapse.show">
-            <a href="{% url 'landing-home' %}" class="nav-link nav-text-color book-button hvr-box-shadow-inset">
-                <i class="fas fa-play  fa-lg"></i> Login</a>
-        </li>
-        {% endif %}
-      </ul>
-  </div>
-</nav>
+    <div id="row">
+
+        <div class="column" Id="body-image">
+
+        </div>
+
+    </div>
+
+    <!-- Logo and Top Navigation Menu: Added a custom style to the navigation-->
+
+    <nav class="navbar bg-color-navbar navbar-expand-sm container-fluid fixed-top ">
+
+        <a href="index.html" class="navbar-brand nav-text-color"></a>
+
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="col-5 ml-auto justify-content-end ">
+            <ul class="navbar-nav mr-auto">
+
+                <a class="navbar-brand" href="{% url 'landing-home' %}" target="show_blank"><span id="logo">logo</span> </a>
+
+                <li class="nav-item mr-2 col-md-3 col-sm-4" data-toggle="collapse" data-target=".navbar-collapse.show">
+                    <a href="index.html" class="nav-link nav-text-color book-button hvr-box-shadow-inset">
+                        <i class="fas fa-info  fa-lg"></i> About</a>
+                </li>
+
+                {% if user.is_authenticated %}
+
+                <li class="nav-item mr-2 col-md-3 col-sm-4" data-toggle="collapse" data-target=".navbar-collapse.show">
+                    <a href="{% url 'landing-home' %}" class="nav-link nav-text-color book-button hvr-box-shadow-inset">
+                        <i class="fas fa-play  fa-lg"></i>Logout</a>
+                </li>
+
+                {% else %}
+                <li class="nav-item mr-2 col-md-3 col-sm-4" data-toggle="collapse" data-target=".navbar-collapse.show">
+                    <a href="{% url 'landing-home' %}" class="nav-link nav-text-color book-button hvr-box-shadow-inset">
+                        <i class="fas fa-play  fa-lg"></i> Sign Up</a>
+                </li>
+
+                <li class="nav-item mr-2 col-md-3 col-sm-4" data-toggle="collapse" data-target=".navbar-collapse.show">
+                    <a href="{% url 'landing-home' %}" class="nav-link nav-text-color book-button hvr-box-shadow-inset">
+                        <i class="fas fa-play  fa-lg"></i> Login</a>
+                </li>
+
+                {% endif %}
+            </ul>
+        </div>
+    </nav>
     </header>
```

Comment:
I can't find the opening <header> tag, it appears to have been removed?

Reply:
Oops! You're right. I'll add the header element opening tag. Thanks.